runtime-config-linux.md

```diff
@@ -146,7 +146,7 @@ For more information, see the [kernel cgroups documentation](https://www.kernel.
 
 The path to the cgroups can be specified in the Spec via `cgroupsPath`.
 `cgroupsPath` is expected to be relative to the cgroups mount point.
-If not specified, cgroups will be created under '/'.
+If not specified or specified as empty string, implementations can define the default cgroup path.
```

Comment:
On Wed, Nov 18, 2015 at 03:22:54AM -0800, Qiang Huang wrote:
> > -If not specified, cgroups will be created under '/'.
> > +If not specified or specified as empty string, implementations can define the default cgroup path.
> 
> @wking Are you suggesting we go back to my original phrasing in this PR?

Yes.

> I think `undefined behavior` is OK, because I think undefined means
> we don't restrict this behavior, doesn't mean the behavior is
> unknown.

It would mean that the spec has nothing to say about what happens when
cgroupsPath is unset, but I think we *do* want to define what happens
when cgroupsPath is unset (and just have the runtime pick its favorite
cgroupsPath name and then carry on as if cgroupsPath was set).